### ISSUE TITLE
Preview not rendering tables and admonitions properly

### ISSUE THREAD

**Issue by gurbindersingh** [asciidoctor/asciidoctor-vscode#516]
## Description

The preview tab is not rendering the tables and admonitions as expected.

### VS Code
Version: 1.64.0 (user setup)
Commit: 5554b12acf27056905806867f251c859323ff7e9
Date: 2022-02-03T04:22:20.678Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19044

### Extension

Version: 2.9.3

## To Reproduce

Steps to reproduce the issue:

**1. Create a file with the following content:**

```adoc
= Document Title

.Table Title
|===
|Column 1, Header Row |Column 2, Header Row

|Cell in column 1, row 1
|Cell in column 2, row 1

|Cell in column 1, row 2
|Cell in column 2, row 2
|===

IMPORTANT: Don't forget the children!

TIP: Look for the warp zone under the bridge.

CAUTION: Slippery when wet.

WARNING: The software you're about to use is untested.

IMPORTANT: Sign off before stepping away from your computer.

[NOTE]
====
An admonition block may contain complex content.

.A list
- one
- two
- three

Another paragraph.
====
```

**2. Open the preview tab**

## Screenshots & Files

**This is how it appears in the preview tab:**

![image](https://user-images.githubusercontent.com/26441580/152676349-133ca6b0-ece6-4cdd-9a56-ffd7a7762d9b.png)

**This is how the rendered HTML looks like:** 

![image](https://user-images.githubusercontent.com/26441580/152676391-14578b53-83d6-48b7-8366-11d0da29c550.png)

**Comment by gurbindersingh** [comment]
## Update

Just after submitting this the extension got an update and the rendering looks like this now 

![image](https://user-images.githubusercontent.com/26441580/152677058-fb634ca5-b1f4-4fca-89b5-f1f7111dc882.png)

**Comment by Mogztter** [comment]
This is a duplicate of #501.
As you've mentioned, this issue is now fixed in version 2.9.4.
